src/index.mjs

```diff
@@ -8,6 +8,8 @@ export { default as ReportFilters } from './lev-report/ReportFilters';
 export { default as Report } from './lev-report/Report';
 export { default as LevReport } from './lev-report';
 export { default as LevUsage } from './lev-report/charts/LevUsage';
+export { default as LevDashboard } from './lev-dashboard';
+export { default as DashboardContent } from './lev-dashboard/DashboardContent';
```

Comment:
Should `./lev-dashboard` export `DashboardContent`?

Is it right to export `DashboardContent`? Does it have any use outside of `LevDashboard`?

Reply:
Yes when we `hydrate` the frontend

Reply:
Don't you just hydrate the `LevDashboard` though? I'd think everything else should come out in the wash.

Reply:
Let's fix this later. (Perhaps when we update lev-react-renderer.)